src/prep/elaboration-core.k

```diff
@@ -182,8 +182,8 @@ when
 syntax KItem ::= haveElabRes ( KListWrap )          [function]
 rule haveElabRes([_,, elabRes(_),, _]) => true
 
-//K bug additional rule required since we introduced KListWrap
-rule haveElabRes([_,, KList2KLabel _,, elabRes(_),, _ (_),, _]) => true
+//K bug additional rule required since we introduced KListWrap? todo (Shijiao)
+//rule haveElabRes([_,, KList2KLabel _,, elabRes(_),, _ (_),, _]) => true
```

Comment:
I think `KList2KLabel` used to be a production, but I can't find it in the current `/include` directory.
In this case, if you know how you want the rule to look, then you should put the types on the variables.